packages/inventory/src/api/index.js

```diff
@@ -72,14 +88,15 @@ export function getEntities(items, {
         return hosts.apiHostGetHostList(
             undefined,
             undefined,
-            hostnameOrId && hostnameOrId.filter,
-            undefined,
+            hostnameOrId,
             undefined,
             undefined,
             perPage,
             page,
             orderBy,
             orderDirection,
+            '', //staleness
```

Comment:
Do you want to keep this comment?

Reply:
Yup, those arguments are starting to get bigger so just to pin point what is what.